File: EventFlowAPI/EventFlowAPI.Logic/DTO/Statistics/ResponseDto/EventStatistics.cs
namespace EventFlowAPI.Logic.DTO.Statistics.ResponseDto
{
    public class EventStatistics
    {
        public int AllAddedEventsCount { get; set; }
        public int AllEventsThatTookPlaceInTimePeriod { get; set; }
        public int AllCanceledEventsCount { get; set; }
        public int DurationAvg { get; set; }
        public double TotalEventsIncome { get; set; }
        public Dictionary<string, double>  MostPopularEvents { get; set; } = [];
        public Dictionary<string, double>  MostProfitableEvents { get; set; } = [];
        public Dictionary<string, double> EventHallDict { get; set; } = [];
        public Dictionary<string, double> EventCategoryDict { get; set; } = [];
    }
}

File: EventFlowClient/src/models/response_models/stats/EventStatistics.ts
export type EventStatistics = {
  allAddedEventsCount: number;
  allEventsThatTookPlaceInTimePeriod: number;
  allCanceledEventsCount: number;
  durationAvg: number;
  totalEventsIncome: number;
  mostPopularEvents: Record<string, number>;
  mostProfitableEvents: Record<string, number>;
  eventHallDict: Record<string, number>;
  eventCategoryDict: Record<string, number>;
};
